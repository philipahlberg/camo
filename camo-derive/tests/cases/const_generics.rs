use camo_derive::Camo;

#[derive(Camo)]
struct Foo<const N: usize> {
    foo: [i32; N],
}

fn main() {}
