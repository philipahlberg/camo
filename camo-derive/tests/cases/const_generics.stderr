error: `camo` does not support const generics
 --> tests/cases/const_generics.rs:4:12
  |
4 | struct Foo<const N: usize> {
  |            ^^^^^
